docs/upgrading_to_v35.0.md

```diff
@@ -0,0 +1,5 @@
+# Upgrading to v35.0
+The v35.0 release of *kubernetes-engine* is a backwards incompatible release.
+
+### Google Cloud Platform Provider upgrade
+The Terraform Kubernetes Engine Module now requires version 6 of the Google Cloud Platform Providers.
```

Comment:
for me, I didn't run into too many actual issues going from v5 => v6, but might be worth including a link to https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_6_upgrade as well as any other docs that end up coming up relating to anything module-specific?

Reply:
SG - Added